From: Richard Purdie <richard.purdie@linuxfoundation.org>
Subject: [PATCH 4/5] gcc/nios2: Define the musl linker

Add a definition of the musl linker used on the nios2 platform.

2021-10-26 Richard Purdie <richard.purdie@linuxfoundation.org>

gcc/ChangeLog:

    * config/nios2/linux.h (MUSL_DYNAMIC_LINKER): Add musl linker

Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>

Submitted [https://gcc.gnu.org/pipermail/gcc-patches/2021-October/582723.html]
Upstream-Status: Backport [https://gcc.gnu.org/git/?p=gcc.git;a=commitdiff;h=e5ddbbf992b909d8e38851bd3179d29389e6ac97]
---
 gcc/config/nios2/linux.h | 1 +
 1 file changed, 1 insertion(+)

Index: gcc-11.2.0/gcc/config/nios2/linux.h
===================================================================
--- gcc-11.2.0.orig/gcc/config/nios2/linux.h
+++ gcc-11.2.0/gcc/config/nios2/linux.h
@@ -30,6 +30,7 @@
 #define CPP_SPEC "%{posix:-D_POSIX_SOURCE} %{pthread:-D_REENTRANT}"
 
 #define GLIBC_DYNAMIC_LINKER "/lib/ld-linux-nios2.so.1"
+#define MUSL_DYNAMIC_LINKER  "/lib/ld-musl-nios2.so.1"
 
 #undef LINK_SPEC
 #define LINK_SPEC LINK_SPEC_ENDIAN \
